symfit/core/minimizers.py

```diff
@@ -20,8 +21,10 @@ def __init__(self, objective, parameters):
         :param objective: Objective function to be used.
         :param parameters: List of :class:`~symfit.core.argument.Parameter` instances
         """
-        self.objective = objective
-        self.params = parameters
+        self._parameters = parameters
```

Comment:
why the underscore here? For fixed_params I would agree, but I don't see why for parameters itself.

Reply:
Meh. Could be just `self.parameters`. I considered requiring it down the line an implementation detail at the time.